<?xml version="1.0" encoding="UTF-8" ?>
<testsuites>
  <testsuite errors="1" failures="0" hostname="air-de-david.lan" id="0" name="UserTests" package="api.users" tests="1" time="0.318" timestamp="2014-08-01T03:55:38">
      <properties />

      <testcase classname="api.users.UserTests" name="test_must_get_UserList_with_userId_is_blank" time="0.317">
          <error message="Cannot get property &apos;status&apos; on null object" type="java.lang.NullPointerException">java.lang.NullPointerException: Cannot get property &apos;status&apos; on null object
	at api.users.UserTests.test_must_get_UserList_with_userId_is_blank(UserTests.groovy:45)
</error>

      </testcase>

      <system-out><![CDATA[--Output from test_must_get_UserList_with_userId_is_blank--
]]></system-out>

      <system-err><![CDATA[--Output from test_must_get_UserList_with_userId_is_blank--
]]></system-err>

  </testsuite>
  <testsuite errors="0" failures="0" hostname="air-de-david.lan" id="1" name="UserControllerTests" package="api.users" tests="1" time="1.369" timestamp="2014-08-01T03:55:27">
      <properties />

      <testcase classname="api.users.UserControllerTests" name="testSomething" time="1.289" />

      <system-out><![CDATA[--Output from testSomething--
]]></system-out>

      <system-err><![CDATA[--Output from testSomething--
]]></system-err>

  </testsuite>
  <testsuite errors="0" failures="0" hostname="air-de-david.lan" id="2" name="UserMagamentServiceTests" package="api.users" tests="1" time="0.225" timestamp="2014-08-01T03:55:28">
      <properties />

      <testcase classname="api.users.UserMagamentServiceTests" name="testSomething" time="0.026" />

      <system-out><![CDATA[--Output from testSomething--
]]></system-out>

      <system-err><![CDATA[--Output from testSomething--
]]></system-err>

  </testsuite>
  <testsuite errors="0" failures="0" hostname="air-de-david.lan" id="3" name="UserTests" package="api.users" tests="6" time="1.24" timestamp="2014-08-01T03:55:28">
      <properties />

      <testcase classname="api.users.UserTests" name="test_UserIsNotValidate_WhitEmailUnique" time="1.048" />

      <testcase classname="api.users.UserTests" name="test_UserIsNotValidate_whitFormatEmailInvalid" time="0.032" />

      <testcase classname="api.users.UserTests" name="test_UserIsNotValidate_whitPasswordLongMinInvalid" time="0.036" />

      <testcase classname="api.users.UserTests" name="test_UserIsNotValidate_whitPasswordLongMaxInvalid" time="0.029" />

      <testcase classname="api.users.UserTests" name="test_UserIsValid_whitStatusActiveByDefaultValue" time="0.019" />

      <testcase classname="api.users.UserTests" name="test_creaUsuario" time="0.068" />

      <system-out><![CDATA[--Output from test_UserIsNotValidate_WhitEmailUnique--
--Output from test_UserIsNotValidate_whitFormatEmailInvalid--
--Output from test_UserIsNotValidate_whitPasswordLongMinInvalid--
--Output from test_UserIsNotValidate_whitPasswordLongMaxInvalid--
--Output from test_UserIsValid_whitStatusActiveByDefaultValue--
--Output from test_creaUsuario--
ocurrio un error, pero cual?org.codehaus.groovy.grails.plugins.testing.GrailsMockErrors: 1 errors
Field error in object 'api.users.User' on field 'sex': rejected value [M]; codes [api.users.User.sex.inList.error.api.users.User.sex,api.users.User.sex.inList.error.sex,api.users.User.sex.inList.error.java.lang.String,api.users.User.sex.inList.error,user.sex.inList.error.api.users.User.sex,user.sex.inList.error.sex,user.sex.inList.error.java.lang.String,user.sex.inList.error,api.users.User.sex.not.inList.api.users.User.sex,api.users.User.sex.not.inList.sex,api.users.User.sex.not.inList.java.lang.String,api.users.User.sex.not.inList,user.sex.not.inList.api.users.User.sex,user.sex.not.inList.sex,user.sex.not.inList.java.lang.String,user.sex.not.inList,not.inList.api.users.User.sex,not.inList.sex,not.inList.java.lang.String,not.inList]; arguments [sex,class api.users.User,M,[Hombre, Mujer]]; default message [Property [{0}] of class [{1}] with value [{2}] is not contained within the list [{3}]]]]></system-out>

      <system-err><![CDATA[--Output from test_UserIsNotValidate_WhitEmailUnique--
--Output from test_UserIsNotValidate_whitFormatEmailInvalid--
--Output from test_UserIsNotValidate_whitPasswordLongMinInvalid--
--Output from test_UserIsNotValidate_whitPasswordLongMaxInvalid--
--Output from test_UserIsValid_whitStatusActiveByDefaultValue--
--Output from test_creaUsuario--
]]></system-err>

  </testsuite>
</testsuites>
